nixos/doc/manual/release-notes/rl-2205.section.md

```diff
@@ -621,6 +621,17 @@ In addition to numerous new and upgraded packages, this release has the followin
   To import that data after to the upgrade, run
   `sudo -u epgstation epgstation run v1migrate /tmp/epgstation.bak`
 
+- How `gdk-pixbuf` modules are set has changed.
+  - `gdk-pixbuf` has been patched to accept multiple `loaders.cache` files in `$GDK_PIXBUF_MODULE_FILE`, separated by `:`
+  - Setting modules through `wrapGAppsHook`: 
+    `wrapGAppsHook` now automatically includes modules from `gdk-pixbuf` and `librsvg`. This is backwards-compatible for most packages.
+    It no longer looks for a module file in `$GDK_PIXBUF_MODULE_FILE` in the build environment, `gdk-pixbuf` no longer sets it when added to `buildInputs`, and you no longer need to add `gdk-pixbuf` or `librsvg` to `buildInputs` unless the program directly depends on it.
+    To include additional gdk-pixbuf modules, set `extraGdkPixbufModules` with an array of packages.
+  - Manually wrapping `$GDK_PIXBUF_MODULE_FILE`:
+    The `librsvg` loaders.cache still contains all of the loaders from `gdk-pixbuf`. This might change in the future, so it is recommended that you explicitly include both.
+    As this change allows users to extend programs with their own loaders, it is recommended that you `--prefix` rather than `--set` `$GDK_PIXBUF_MODULE_FILE` with your values.
+    For example: `makeWrapper ... --prefix GDK_PIXBUF_MODULE_FILE : "${librsvg}/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache:${gdk-pixbuf}/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache"`
```

Comment:
Maybe also note something like “But using `wrapGAppsHook`/`wrapGAppsNoGuiHook` with `extraGdkPixbufModules` is preferred.” to be safe.